src/metrics/metrics.md

```diff
@@ -2,10 +2,59 @@
 
 Here we provide definitions and information about our standard metrics.
 
-Table of contents with links to all metrics.
+## Table of Contents
+
+<!-- toc -->
 
 -----
 
+## Daily Active Users (DAU)
+
+### Overview
+
+DAU counts the number of unique profiles active in the product on each day.  This is intended to approximate the number of people using the product each day.
```

Comment:
We should define with a profile is in terminology (https://docs.telemetry.mozilla.org/concepts/terminology.html)